src/fn_call.rs

```diff
@@ -622,11 +622,11 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
                 let name = this.read_scalar(args[0])?.to_i32()?;
 
                 trace!("sysconf() called with name {}", name);
-                // Cache the sysconf integers via Miri's global cache.
```

Comment:
We don't currently cache this, do we? The comment makes it sound like we do but I found no such code.